setup.cfg

```diff
@@ -91,14 +6,11 @@ max-line-length = 95
 max-line-length = 95
 copyright-check = True
 copyright-author = MetPy Developers
-inline-quotes = single
-multiline-quotes = double
 rst-roles = class, data, doc, func, meth, mod
 rst-directives = plot, versionchanged
-docstring-convention = numpy
 known-modules = matplotlib:[matplotlib,mpl_toolkits],netcdf4:[netCDF4]
 exclude = docs build src/metpy/io/_metar_parser/metar_parser.py
-select = A B C D E F G H I J K L M N O P Q R S T U V W X Y Z B902
+select = C I R
 ignore = F405 W503 RST902 SIM106
 per-file-ignores = examples/*.py: D MPY001 T003 T201
                    tutorials/*.py: D MPY001 T003 T201
```

Comment:
Only can remove the T* rules (which I've done now since they were flake8-print). The remaining files all have rules from plugins we're keeping at the moment.